includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -2562,4 +2563,48 @@ private function get_fingerprint_metadata(): array {
 
 		return $customer_fingerprint_metadata;
 	}
+
+	/**
+	 * List authorizations
+	 *
+	 * @param int    $page       The requested page.
+	 * @param int    $page_size  The size of the requested page.
+	 * @param string $sort       The column to be used for sorting.
+	 * @param string $direction  The sorting direction.
+	 *
+	 * @return array
+	 * @throws API_Exception - Exception thrown on request failure.
+	 */
+	public function list_authorizations( $page = 0, $page_size = 25, $sort = 'created', $direction = 'desc' ) {
```

Comment:
The methods of this class are tested in `tests/unit/wc-payment-api/test-class-wc-payments-api-client.php`. I'd suggest to add some test cases there for the new methods we are adding. What do you think?

Reply:
I was trying to find where tests are written but couldn't. Thank you for the help, I'll work on tests now!